spec/helpers/alert_notifier_helper_spec.rb

```diff
@@ -147,4 +147,52 @@
       end
     end
   end
+
+  describe "#subject" do
+    let(:alert) { create(:alert, address: "123 Sample St") }
+    let(:application) do
+      mock_model(Application, address: "Bar Street",
+      description: "Alterations & additions", council_reference: "007",
+      location: double("Location", lat: 1.0, lng: 2.0))
+    end
+    let(:comment) { create(:comment, application: application) }
+
+    context "with an application" do
+      subject { helper.subject(alert, [application], []) }
+      it { should eql "1 new planning application near 123 Sample St" }
+    end
+
+    context "with a comment" do
+      subject { helper.subject(alert, [], [comment]) }
```

Comment:
I think you could consider using [keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments) on this method, at the moment you need to look around at other tests to understand what could have been in the empty array.